package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "restaurant-homepage",
+  "version": "1.0.0",
+  "description": "It is a resturant homepage which shows it's menu build by using js.",
+  "private": true,
+  "scripts": {
+    "build": "webpack",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Comment:
Great work adding this dependency. However, let's improve it and make it better. You can add other dependencies such as start

`"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
`